regolith/helpers/a_expensehelper.py

```diff
@@ -103,81 +118,61 @@ def db_updater(self):
         else:
             pdoc = {}
 
+
         pdoc.update({
-            'begin_date': now,
-            'log_url': '',
-            'name': rc.name,
-            'pi_id': rc.pi_id,
-            'lead': rc.lead,
-        })
-        if rc.lead is "tbd":
-            pdoc.update({
-                'status': 'proposed'
-            })
+            "begin_year": int(begin_date.split('-')[0]),
+             "begin_month": int(begin_date.split('-')[1]),
+             "begin_day": int(begin_date.split('-')[2]),
+             "end_year": int(end_date.split('-')[0]),
+             "end_month": int(end_date.split('-')[1]),
+             "end_day": int(end_date.split('-')[2])
+             })
+        if rc.business:
+            pdoc.update({'expense_type': "business"
+                         })
         else:
-            pdoc.update({
-                'status': 'started'
-            })
-
-        if rc.description:
-            pdoc.update({
-                'description': rc.description,
-            })
+            pdoc.update({"expense_type":"travel"
+                         })
+    # Updates the grants, assumes equal percentage for each grant
         if rc.grants:
             if isinstance(rc.grants, str):
                 rc.grants = [rc.grants]
-            pdoc.update({'grants': rc.grants})
+            percentages = [100 / len(rc.grants) for i in rc.grants]
         else:
-            pdoc.update({'grants': ["tbd"]})
-        if rc.group_members:
-            if isinstance(rc.group_members, str):
-                rc.group_members = [rc.group_members]
-            pdoc.update({'group_members': rc.group_members})
+            rc.grants = 'tbd'
+            percentages = 0 #No grant => no percentage?
+
+        # segregated expense
+        if not rc.segregated:
+            rc.segregated = "0"
```

Comment:
I think this should be an int not str

Reply:
I will convert this to an int, (I was converting it to an int at a later stage).

Reply:
It may be better to specify a default in the parser, then you don't need this if statement.

Reply:
done